test/navigation/navigators/MatchingRoutes.scala

```diff
@@ -57,15 +58,34 @@ trait MatchingRoutes {
           }
         }
 
-        "go to TrustName from WhatIsTheUTR" in {
-          forAll(arbitrary[UserAnswers]) {
-            userAnswers =>
+        "user is an organisation" must {
 
-              navigator.nextPage(WhatIsTheUTRPage, NormalMode)(userAnswers)
-                .mustBe(routes.TrustNameController.onPageLoad(NormalMode))
+          "go to TrustName from WhatIsTheUTR" in {
+            forAll(arbitrary[UserAnswers]) {
+              userAnswers =>
+
+                navigator.nextPage(WhatIsTheUTRPage, NormalMode)(userAnswers)
+                  .mustBe(routes.TrustNameController.onPageLoad(NormalMode))
```

Comment:
This test should be failing as you've got in the implementation to go to AgentClientReference.